File: dhandho-jsclient/src/app/app.module.ts
import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import { FormsModule } from '@angular/forms';
import { HttpModule } from '@angular/http';


import { AppComponent } from './app.component';
import { CommandComponent } from './command/command.component';


@NgModule( {
    declarations: [
        AppComponent,
        CommandComponent
    ],
    imports: [
        BrowserModule, FormsModule, HttpModule
    ],
    providers: [],
    bootstrap: [AppComponent]
} )
export class AppModule { }

File: dhandho-jsclient/src/main/java/app/dhojsw/ng/AppModule.java
package app.dhojsw.ng;

import app.dhojsw.ng.command.CommandComponent;
import app.dhojsw.ng.my.MyComponent;
import app.dhojsw.ng.service.Logger;
import app.dhojsw.ng.util.RouterModuleForRoot;
import def.angular.common_http.HttpClientModule;
import def.angular.core.NgModule;
import def.angular.forms.FormsModule;
import def.angular.platform_browser.BrowserModule;

@NgModule(//
		imports = { BrowserModule.class, FormsModule.class, HttpClientModule.class, RouterModuleForRoot.class }, //
		declarations = { AppComponent.class, MyComponent.class, CommandComponent.class, PageNotFoundComponent.class }, //
		bootstrap = { AppComponent.class, CommandComponent.class }, //
		providers = { Logger.class }//
)

public class AppModule {

}